onnxruntime/core/providers/openvino/backends/basic_backend.cc

```diff
@@ -144,8 +144,8 @@ void BasicBackend::PopulateConfigValue(ov::AnyMap& device_config) {
     device_config.emplace(ov::hint::inference_precision("f32"));
   }
   if (global_context_.precision_str.find("ACCURACY") != std::string::npos &&
-      global_context_.device_type == "GPU") {
-    if (global_context_.OpenVINO_Version.at(0) >= 2024 && global_context_.OpenVINO_Version.at(1) >= 1) {
+      global_context_.device_type.find("GPU") != std::string::npos) {
+    if (((OPENVINO_VERSION_MAJOR == 2024) && (OPENVINO_VERSION_MINOR > 1)) || (OPENVINO_VERSION_MAJOR > 2024)) {
```

Comment:
@jatinwadhwa921 
I think we should not be using "OPENVINO_VERSION_MAJOR" etc. to check the version
Instead use :
"global_context_.OpenVINO_Version.at(0)" etc.
I think the former is a macro available at compile time and we will get warnings during build because the condition evaluates to a constant.
I just built it with this change and got the following build failure on this line (since warnings are treated as errors)  

"warning C4127: conditional expression is constant"